go/packages/dev-mode/main_test.go

```diff
@@ -39,7 +39,7 @@ type ValidationResult struct {
 }
 
 var port = 9001
-var region = "us-east-1"
+var region = "us-east-2"
```

Comment:
I will switch this back to us-east-1 once the telemetry API is rolled out in all regions 😎